src/parser.rs

```diff
@@ -609,6 +610,15 @@ pub fn initialize() -> Command {
                 .args(["debug_log", "output"])
                 .multiple(true),
         )
+        .arg(
+            Arg::new("update_app")
+                .short('U')
+                .long("update")
+                .conflicts_with_all(["url", "wordlist"])
```

Comment:
```suggestion
                .exclusive(true)
```
let's just make it incompatible with every other option